src/transformers/models/deberta_v2/modeling_deberta_v2.py

```diff
@@ -753,8 +753,6 @@ def disentangled_attention_bias(self, query_layer, key_layer, relative_pos, rel_
                 r_pos = relative_pos
 
             p2c_pos = torch.clamp(-r_pos + att_span, 0, att_span * 2 - 1)
-            if query_layer.size(-2) != key_layer.size(-2):
```

Comment:
All 

      ``` self.query_proj = nn.Linear(config.hidden_size, self.all_head_size, bias=True)
        self.key_proj = nn.Linear(config.hidden_size, self.all_head_size, bias=True)
        self.value_proj = nn.Linear(config.hidden_size, self.all_head_size, bias=True)```

have the same dim which is why this control flow is never true no? 

LGTM to remove for me

Reply:
> have the same dim which is why this control flow is never true no?

Yes

Reply:
`Deberta-v2` same dim always. so the control flow is never true